daemon/inertia/webhook.go

```diff
@@ -13,11 +13,13 @@ import (
 	git "gopkg.in/src-d/go-git.v4"
 )
 
+var webhookSecret = "inertia"
```

Comment:
Add a comment to describe global variables?

Reply:
This variable will likely deprecated by a branch I'm working on, so I'll leave it for now